include/arch/x86/processor.h

```diff
@@ -241,7 +241,21 @@ typedef uint64_t x86_reg_t;
 typedef uint32_t x86_reg_t;
 #endif
 
-struct cpu_regs {
+struct __packed cpu_exc {
```

Comment:
Move `__packed` after the closing bracket, like:
```c
struct cpu_exc {
   ...
} __packed;
```

I see that we use both styles in KTF but the above seems to have more users and we should settle on one style only, imho.